spec/serializers/trainee_serializer/v01_spec.rb

```diff
@@ -38,13 +38,9 @@
         trainee_start_date
         training_initiative
         hesa_id
-        ethnicity
```

Comment:
I think the docs and data dictionary currently use the field name `ethnicity` to map the HESA ethnicity field, so we need to look at this further to work what should change.